### DISCUSSION TITLE
Port Reporting Issue

### DISCUSSION THREAD

**Discussion by Exergist** [jchristn/WatsonWebsocket#113] [General]
I encountered some kind of issue with reporting the port via the `ConnectionEventArgs`. 

In this example I targeted `127.0.0.1` on port `4548`. 

Upon client connection the (server) `ConnectionEventArgs` contains:

![image](https://user-images.githubusercontent.com/31357974/211182810-d523f88b-06b3-4877-b93d-e4034b4f1ceb.png)

...however the client's contents DO seem to show the correct target:

![image](https://user-images.githubusercontent.com/31357974/211182774-c82704c9-59a4-4547-8ea5-beda6a8d9b5a.png)

Each time I run my code the server-reported `args.Client.Port` changes to a new number, seemingly above 50000. 

Also note that this same issue seems to occur for the `DisconnectionEventArgs`.

**Any thoughts as to what might be going on or what I might be missing?**

**Comment by jchristn** [comment]
This is expected. Client port numbers are generally random over a certain number and chosen by the operating system. The server port is static and defined by you.

**Reply by Exergist** [reply]
Thanks for the feedback!
